### ISSUE TITLE
Helm chart: Create admin user

### ISSUE THREAD

**Issue by axisofentropy** [UffizziCloud/uffizzi_app#113]
**Tell us about your request**
Currently, open-source Uffizzi users must run `rake uffizzi_core:create_user` to create the first user after installing Uffizzi using Helm. I believe Helm can do this for them.

**Which service(s) is this request for?**
open-source Helm chart

**Describe the solution you'd like**

- The Helm chart should recognize `global` values for the username, password, and project name.
- IF the username and password are specified, the Helm chart should specify a `Pod` and/or `Job` that executes the `create_user` rake task after all installation is complete (and `postgres` is running.) It should retry until it succeeds, and then be deleted.
- This `Job` and/or `Pod` should be created upon after every new release, using Helm hooks. https://helm.sh/docs/topics/charts_hooks/

**Are you currently working around the issue?**
We will probably document how to execute this command using `kubectl` to make it easier until this is implemented.

**Additional context**
This builds upon #30

**Comment by sergeykirillovdb** [comment]
Yeah, helm has a post-install hook and we can use it. But we discussed it with Anton and decided to use Ruby application load commands. It must be simpler. That's why I'm working in this way

**Comment by sergeykirillovdb** [comment]
For QA

First of all, I fixed the function for the admin user creating. It can be executed everytime when application starts
Pull last 'feature/113_helm_create_user' branch (open source version from github) to your local machine

### Test cases

1. User created successfully
a) `docker-compose run --rm web bash -c "rake uffizzi_core:create_user"`
b) press enter on "Enter User Email" if you don't have user with the default email or enter another email (I use default user@example.com)
c) enter any non-empty password
d) press enter on "Enter Project Name"
e) check console is ready for new commands
f) check user created

2. **User not created through the console if it has already exists**
Steps to check
a) `docker-compose run --rm web bash -c "rake uffizzi_core:create_user"`
b) enter email of the existing user (user@example.com)
c) check the error like "User already exists. Abort user creating"
d) check console is ready for new commands

3. **User not created through the console if there are no password**
Steps to check
a) `docker-compose run --rm web bash -c "rake uffizzi_core:create_user"`
b) enter any new email (not user@example.com)
c) press enter on "Enter Password"
d) check the error like "Password can't be blank. Abort user creating"
e) check console is ready for new commands

4. **User not created through the console with env param if it has already exists**
Steps to check
a) `docker-compose run --rm -e UFFIZZI_USER_EMAIL=user@example.com  -e UFFIZZI_USER_PASSWORD=password -e UFFIZZI_PROJECT_NAME=project web bash -c "rake uffizzi_core:create_user"`
b) check there are no errors
c) check console is ready for new commands

5. **Application up well if there are problems with default user data**
a) Edit command for the web service in the "docker-compose.yml" file (near to the 35 string now)
   `command: bash -c "bundle install && bundle exec rails db:create db:migrate && bundle exec rake uffizzi_core:create_user && bundle exec puma -C config/puma.rb"`
b) `docker-compose up`
c) check application is working (`web_1           | * Listening on tcp://0.0.0.0:7000` in the console logs is ok)

6. **User created successfully if it's the first compose launch**
a) Like in the 5. case, edit command for the web service in the "docker-compose.yml" file (near to the 35 string now)
   `command: bash -c "bundle install && bundle exec rails db:create db:migrate && bundle exec rake uffizzi_core:create_user && bundle exec puma -C config/puma.rb"`
b)  Add several variables to the "x-web-environment" section 
```
UFFIZZI_USER_EMAIL: user-new@example.com
UFFIZZI_USER_PASSWORD: password
UFFIZZI_PROJECT_NAME: default
```
c) `docker-compose up`
d) check application is working
e) check that user with email "user-new@example.com" exists

7. **User not created if there are problems with email**
a) Like in the 5. case, edit command for the web service in the "docker-compose.yml" file (near to the 35 string now)
   `command: bash -c "bundle install && bundle exec rails db:create db:migrate && bundle exec rake uffizzi_core:create_user && bundle exec puma -C config/puma.rb"`
b)  Add several variables to the "x-web-environment" section 
```
UFFIZZI_USER_EMAIL: user-error-email@example.com"
UFFIZZI_USER_PASSWORD: password
UFFIZZI_PROJECT_NAME: default
```
c) `docker-compose up`
d) check application is working
e) check that user with email "user-error-email@example.com" doesn't exist

8. **User not created if there are empty password**
a) Like in the 5. case, edit command for the web service in the "docker-compose.yml" file (near to the 35 string now)
   `command: bash -c "bundle install && bundle exec rails db:create db:migrate && bundle exec rake uffizzi_core:create_user && bundle exec puma -C config/puma.rb"`
b)  Add several variables to the "x-web-environment" section 
```
UFFIZZI_USER_EMAIL: user-empty-password@example.com
UFFIZZI_USER_PASSWORD: ""
UFFIZZI_PROJECT_NAME: default
```
c) `docker-compose up`
d) check application is working
e) check that user with email "user-empty-password@example.com" doesn't exist


### Remove all changes from the code after the QA
`git restore ./`

**Comment by sergeykirillovdb** [comment]
Second one about helm installation
I'm not sure about free GKE, AKS and EKS, so you can try to use local minikube. It looks like you can check the installation with it
1. https://artifacthub.io/packages/helm/uffizzi-app/uffizzi-app#example-helm-values-file
2. make small change to the install command `helm install my-uffizzi-app charts/uffizzi-app/ -f values.yaml --namespace uffizzi --create-namespace`
3. `kubectl get pods -n uffizzi`
4. `kubectl exec -it -n uffizzi-local my-uffizzi-app-web-6f64b9b97b-kl968 bash`
5. `rails c`
6. check there are user with the email from the "values.yaml" `user = UffizziCore::User.first`
7. check password `user.authenticate(your_password)`

**Comment by NealArw** [comment]
Waiting for setting up cluster.

**Comment by sergeykirillovdb** [comment]
Made setup on the AWS "[install-training-2022-05](https://us-east-2.console.aws.amazon.com/eks/home?region=us-east-2#/clusters/install-training-2022-05)"
URL for the https://helm-test.app.qa-gke.uffizzi.com/

Cases

- [ ] First install
- [ ] Uninstall and install once again. The user already exists because uninstall doesn't remove database

User and default project exists
<img width="762" alt="Снимок экрана 2022-06-02 в 14 09 51" src="https://user-images.githubusercontent.com/101560154/171607600-1ffe166d-a40e-4c0f-9fbd-364c9e4f551c.png">

Request to the session with the data from helm values is ok
<img width="487" alt="Снимок экрана 2022-06-02 в 14 07 33" src="https://user-images.githubusercontent.com/101560154/171607357-db0307b2-3f4b-4427-8dfc-b73f7d791688.png">

**Comment by axisofentropy** [comment]
I've added another commit that changed the value names and bumped the Helm Chart version number. This all looks good; I approve.

I also successfully tested the common use case after this commit:
![image](https://user-images.githubusercontent.com/66315/171680275-ace386b8-950f-4349-83aa-d4d560a120ce.png)
